tests/web/web_test.bats

```diff
@@ -0,0 +1,70 @@
+# PURPOSE:
+# I will test the following commands: web create, get, list, update, delete, and clean.
+load ../test_setup.bash
+
+# constants
+FILES=$BATS_TEST_DIRNAME/test_files/*
+
+@test "web create" { # web create test
+  for f in $FILES
+  do
+    run $NIM web create "$f"
+    assert_success
+    assert_output --partial "done"
+  done
+}
+
+@test "web get" { # web get test
+  for f in $FILES
+  do
+    filename=$(basename $f)
+    run $NIM web get -p $filename
+    assert_success
+    assert_output --partial "done"
+    assert_output --partial "getting $filename"
+    assert_output --partial "$(cat $f)" # checking if file content is correct
+  done
+}
+
+@test "web list" { # web list test, 404.html is default in namespaces
+  listString1="404.html"$'\n'"test1.html"
+  listString2="test1.html"$'\n'"404.html"
+
+  run $NIM web list
+  assert_success
+  if [ "$listString1" == "$output" ]; then
```

Comment:
Is this logic statement because the order of the lists returned is non-deterministic? We could maybe do something with "sort" in bash to make this simpler (sort output and then compare) but it if gets too complex - just leave it.

Reply:
Gave this a try with a sort method, but with 404.html always being on the list, it was hard to sort cause I had to keep hardcoding it in for some reason. If you have another suggestion, please let me know.